packages/teams-js/src/private/appEntity.ts

```diff
@@ -82,6 +82,11 @@ export namespace appEntity {
     sendMessageToParent('appEntity.selectAppEntity', [threadId, categories, subEntityId], callback);
   }
 
+  /**
+   * Checks if appEntity capability is supported currently
```

Comment:
For consistency ([example](https://github.com/OfficeDev/microsoft-teams-library-js/blob/d7defd384e4bac546d0122c40528a299c7ac5046/packages/teams-js/src/public/dialog.ts#L177)) how about: Checks if appEntity is supported by the host